cartridge/issues.lua

```diff
@@ -344,6 +349,44 @@ local function list_on_instance(opts)
         end
     end
 
+    -- Vshard issues
+    do
+        local vshard = package.loaded.vshard
+        if not vshard then
+            goto continue
+        end
+
+        local routers = vshard.router.internal.routers
+        if routers == nil or next(routers) == nil then
+            goto continue
+        end
+
+        for _, router in pairs(routers) do
+            local alerts = router:info().alerts
+            for _, alert in ipairs(alerts) do
+                local type, message = alert[1], alert[2]
+                log.info(require('yaml').encode(alert))
+                if type == 'UNKNOWN_BUCKETS'
```

Comment:
With this predicate, a cluster shows issues even before the vshard is bootstrapped, and it's no good, I think. Also, WebUI tests fail because of it.